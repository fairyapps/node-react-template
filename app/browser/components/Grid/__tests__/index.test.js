import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";
import { Container, Row, Column } from "../index";

test("Column renders correct snapshot", () => {
  const component = renderer.create(<Column md={1 / 2}>sometext</Column>);
  let tree = component.toJSON();
  expect(tree).toMatchStyledComponentsSnapshot();
});
