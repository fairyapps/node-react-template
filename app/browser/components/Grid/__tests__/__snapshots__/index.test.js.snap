// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Column renders correct snapshot 1`] = `
.jlPgpE {
  box-sizing: border-box;
  vertical-align: middle;
  padding-left: 10px;
  padding-right: 10px;
  margin-bottom: 20px;
  width: 100%;
}

@media (min-width: 992px) {
  .jlPgpE {
    -webkit-flex: 0 0 50%;
    -ms-flex: 0 0 50%;
    flex: 0 0 50%;
    max-width: 50%;
  }
}

<div
  className="sc-htpNat jlPgpE"
>
  sometext
</div>
`;
